### DISCUSSION TITLE
When is it planned to add "Create teams" feature

### DISCUSSION THREAD

**Discussion by web265p3** [pnp/pnpcore#1203] [General] [answered]
When I try to create a new team with the Pnp Core SDK, currently I get an exception "Coming soon..."

Is there any roadmap available when this feature will be added? I think its quite essential.
Also the [docs](https://pnp.github.io/pnpcore/using-the-sdk/teams-intro.html) do not mention anything about the creation of teams:

**Comment by jansenbe** [comment, marked as answer]
@web265p3 : you can today create a group connected Team site and then "teamify" it, see https://pnp.github.io/pnpcore/using-the-sdk/admin-sharepoint-modernize.html#create-a-team-for-a-microsoft-365-group-connected-site. Alternatively you can create a Team via the Teams graph APIs (https://learn.microsoft.com/en-us/graph/api/team-post?view=graph-rest-1.0&tabs=http) and our model to easily make API calls (https://pnp.github.io/pnpcore/using-the-sdk/basics-customapirequests.html#making-a-custom-microsoft-graph-request).  When you do it like this then do know that the SharePoint site is not there immediately, it's created async / on demand.
